Product Management Service/Code/product-service/src/main/java/com/shopkeeper/productservice/dao/ProductServiceRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.shopkeeper.productservice.dao;
+
+import org.springframework.data.repository.CrudRepository;
+
+import com.shopkeeper.productservice.dto.Product;
+
+public interface ProductServiceRepository extends CrudRepository<Product, Integer>{
```

Comment:
What do the types inside the CrudRepository represent?

Reply:
Why CrudRepository, why not JPARepository? What is the difference and which is more suitable here?